embedchain/embedchain.py

```diff
@@ -179,7 +179,7 @@ def query(self, input_query, config: QueryConfig = None):
             config = QueryConfig()
         context = self.retrieve_from_database(input_query)
         prompt = self.generate_prompt(input_query, context, config.template)
-        answer = self.get_answer_from_llm(prompt)
+        answer = self.get_llm_model_answer(prompt, config.temperature, config.max_tokens)
```

Comment:
just pass the whole config and in the `get_llm_model_answer` method use `config.temperature` and `config.max_tokens`